myhpi/core/templates/core/information_page.html

```diff
@@ -16,6 +16,12 @@ <h1 class="page-title">
     </div>
     <div class="col-lg-3">
         <div class="side-panel-container">
+            {% get_language_info for LANGUAGE_CODE as lang %}
+            {% if lang.code == "en" %}
+                <aside class="side-panel border-accent">
+                    This has page has been machine-translated.
```

Comment:
```suggestion
                    This page may have been machine-translated.
```

Reply:
I think this would be better since other Gremien may actually manually translate it.

Also we could think about explaining it better. Why is it machine translated? What can I do about it?

Reply:
I agree that for a user, an explicit "is" would be more valuable.With the current specification of the issue I believe "may" is the best solution. This means I will be alerted to possible weirdness and know I will ask if something is unclear.

But if we expand the scope we can also have the is / is not . But I think this definitely needs to be set at the time of translation, automatically. So it may be more complicated, and I think this version would already be helpful (though the note may need to be expanded to explain a little better why the hint is therek

Reply:
> But I think this definitely needs to be set at the time of translation

Yes, this should be set when translating a page

> automatically

Only if it it still possible to change this. Otherwise, it would not be possible to machine-translate a page, review it (without any need for changes) and then publish it without a warning.

Reply:
> Only if it it still possible to change this.  Otherwise, it would not be possible to machine-translate a page, review it (without any need for changes) and then publish it without a warning.

Agreed. The question remains whether this PR (in some form, e.g. adapted wording ) is useful or not.